docs/_includes/sidebar.html

```diff
@@ -1,72 +1,76 @@
-<div class="sidebar">
-    <div class="site-name">
-        <h1>{{ site.title }}</h1>
-        <a class="github-button" href="https://github.com/zalando/nakadi"
-           data-size="large" aria-label="GitHub">GitHub</a>
-    </div>
+<div class="sidebar" tabindex="0">
+    <div class="content">
+        <div class="site-name">
+            <span class="menu show">Menu</span>
+            <span class="menu hide" tabindex="0">Menu</span>
+            <a href="index.html"><h1>{{ site.title }}</h1></a>
+
+            <a class="github-button" href="https://github.com/zalando/nakadi"
+               data-size="large" aria-label="GitHub">GitHub</a>
+        </div>
 
-    {% assign allData = site.data %}
+        {% assign allData = site.data %}
```

Comment:
All answers here https://github.com/zalando/nakadi/blob/master/docs/README.md
This is Ruby based https://jekyllrb.com/